NitroxModel/Logger/Log.cs

```diff
@@ -193,6 +198,28 @@ private static string GetLoggerName()
             return name.IndexOf("server", StringComparison.InvariantCultureIgnoreCase) >= 0 ? "Server" : name;
         }
 
+        private static string GetLogFileName()
+        {
+            bool IndexOfAny(string haystack, params string[] needles)
+            {
+                foreach (string needle in needles)
+                {
+                    if (haystack.IndexOf(needle, StringComparison.OrdinalIgnoreCase) >= 0)
+                    {
+                        return true;
+                    }
+                }
+                return false;
+            }
+
+            return GetLoggerName() switch
+            {
+                { } s when IndexOfAny(s, "server") => "server",
+                { } s when IndexOfAny(s, "launcher") => "launcher",
+                _ => "game"
+            };
+        }
```

Comment:
Too much cognitive complexity - why not a simple if/elseif?

Reply:
`foreach` was unnecessary. Just liked expression syntax more myself. Used if else now.